packages/core/src/predicate.ts

```diff
@@ -31,12 +31,35 @@ const predicates: Record<string, PredicateFactory> = {
       }
       return result
     }
+  },
+
+  $like: (query: string, propertyKey: string): Predicate => {
+    const likeSymbol = '*'
```

Comment:
Should it be '%'?

Reply:
I can replace it, but I think '*' is be better here

Reply:
Let's @z2sx to decide.

Reply:
Could you please reduce complexity of new code, please use CodeMetrics to estimations.